ui/components/multichain/account-overview/account-overview-layout.tsx

```diff
@@ -128,6 +128,7 @@ export const AccountOverviewLayout = ({
           onSlideClick={handleCarouselClick}
           onSlideClose={handleRemoveSlide}
           onRenderSlides={handleRenderSlides}
+          className="mb-3 px-4"
```

Comment:
Adding margin and padding relative to where the carousel is placed using tailwind css